### PR TITLE
Release 1.2.0b2

### PR DESCRIPTION
**Pull request by kasteph**
### Added

- Added support for multiple-constraint direct origin dependencies with the same
  version ([#5715](https://github.com/python-poetry/poetry/pull/5715))
- Added support disabling TLS verification for custom package sources via `poetry config certificates.<repository>.cert false` ([#5719](https://github.com/python-poetry/poetry/pull/5719)
- Added new configuration (`virtualenvs.prompt`) to customize the prompt of the Poetry-managed virtual environment ([#5606](https://github.com/python-poetry/poetry/pull/5606))
- Added progress indicator to `download_file` (used when downloading dists) ([#5451](https://github.com/python-poetry/poetry/pull/5451))
- Added `--dry-run` to `poetry version` command ([#5603](https://github.com/python-poetry/poetry/pull/5603))
- Added `--why` to `poetry show` ([#5444](https://github.com/python-poetry/poetry/pull/5444))
- Added support for single page (html) repositories ([#5517](https://github.com/python-poetry/poetry/pull/5517))
- Added support for PEP 508 strings when adding
  dependencies via `poetry add` command ([#5554](https://github.com/python-poetry/poetry/pull/5554))
- Added `--no-cache` as a global option ([#5519](https://github.com/python-poetry/poetry/pull/5519))
- Added cert retrieval for HTTP requests made by Poetry ([#5320](https://github.com/python-poetry/poetry/pull/5320))
- Added `--skip-existing` to `poetry publish` ([#2812](https://github.com/python-poetry/poetry/pull/2812))
- Added `--all-extras` to `poetry install` ([#5452](https://github.com/python-poetry/poetry/pull/5452))
- Added new `poetry self` sub-commands to manage plugins and/or system environment packages, eg: keyring backends ([#5450](https://github.com/python-poetry/poetry/pull/5450))
- Added new configuration (`installer.no-binary`) to allow selection of non-binary distributions when installing a dependency ([#5609](https://github.com/python-poetry/poetry/pull/5609))

### Changed

- `poetry plugin` commands are now deprecated in favor of the more generic `poetry self`
  commands ([#5450](https://github.com/python-poetry/poetry/pull/5450))
- When creating new projects, Poetry no longer restricts README extensions to `md` and `rst` ([#5357](https://github.com/python-poetry/poetry/pull/5357))
- Changed the provider to allow fallback to installed packages ([#5704](https://github.com/python-poetry/poetry/pull/5704))
- Solver now correctly handles and prefers direct reference constraints (vcs, file etc.) over public version identifiers ([#5654](https://github.com/python-poetry/poetry/pull/5654))
- Changed the build script behavior to create an ephemeral build environment when a build script is
  specified ([#5401](https://github.com/python-poetry/poetry/pull/5401))
- Improved performance when determining PEP 517 metadata from sources ([#5601](https://github.com/python-poetry/poetry/pull/5601))
- Project package sources no longer need to be redefined as global repositories when configuring credentials ([#5563](https://github.com/python-poetry/poetry/pull/5563))
- Replaced external git command use with dulwich, in order to force the legacy behaviour set `experimental.system-git-client` configuration to `true` ([#5428](https://github.com/python-poetry/poetry/pull/5428))
- Improved http request handling for sources and multiple paths on same netloc ([#5518](https://github.com/python-poetry/poetry/pull/5518))
- Made `no-pip` and `no-setuptools` configuration explicit ([#5455](https://github.com/python-poetry/poetry/pull/5455))
- Improved application logging, use of `-vv` now provides more debug information ([#5503](https://github.com/python-poetry/poetry/pull/5503))
- Renamed implicit group `default` to `main` ([#5465](https://github.com/python-poetry/poetry/pull/5465))
- Replaced in-tree implementation of `poetry export`
  with `poetry-plugin-export` ([#5413](https://github.com/python-poetry/poetry/pull/5413))
- Changed the password manager behavior to use a `"null"` keyring when
  disabled ([#5251](https://github.com/python-poetry/poetry/pull/5251))
- Incremental improvement of Solver performance ([#5335](https://github.com/python-poetry/poetry/pull/5335))
- Newly created virtual environments on macOS now are excluded from Time Machine backups ([#4599](https://github.com/python-poetry/poetry/pull/4599))
- Poetry no longer raises an exception when a package is not found on PyPI ([#5698](https://github.com/python-poetry/poetry/pull/5698))
- Update `packaging` dependency to use major version 21, this change forces Poetry to drop support for managing Python 2.7 environments ([#4749](https://github.com/python-poetry/poetry/pull/4749))

### Fixed

- Fixed `poetry update --dry-run` to not modify `poetry.lock` ([#5718](https://github.com/python-poetry/poetry/pull/5718), [#3666](https://github.com/python-poetry/poetry/issues/3666), [#3766](https://github.com/python-poetry/poetry/issues/3766))
- Fixed [#5537](https://github.com/python-poetry/poetry/issues/5537) where export fails to resolve dependencies with more than one
  path ([#5688](https://github.com/python-poetry/poetry/pull/5688))
- Fixed an issue where the environment variables `POETRY_CONFIG_DIR` and `POETRY_CACHE_DIR` were not being respected ([#5672](https://github.com/python-poetry/poetry/pull/5672))
- Fixed [#3628](https://github.com/python-poetry/poetry/issues/3628) and [#4702](https://github.com/python-poetry/poetry/issues/4702) by handling invalid distributions
  gracefully ([#5645](https://github.com/python-poetry/poetry/pull/5645))
- Fixed an issue where the provider ignored subdirectory when merging and improve subdirectory support for vcs
  deps ([#5648](https://github.com/python-poetry/poetry/pull/5648))
- Fixed an issue where users could not select an empty choice when selecting
  dependencies ([#4606](https://github.com/python-poetry/poetry/pull/4606))
- Fixed an issue where `poetry init -n` crashes in a root directory ([#5612](https://github.com/python-poetry/poetry/pull/5612))
- Fixed an issue where Solver errors arise due to wheels having different Python
  constraints ([#5616](https://github.com/python-poetry/poetry/pull/5616))
- Fixed an issue where editable path dependencies using `setuptools` could not be correctly installed ([#5590](https://github.com/python-poetry/poetry/pull/5590))
- Fixed flicker when displaying executor operations ([#5556](https://github.com/python-poetry/poetry/pull/5556))
- Fixed an issue where the `poetry lock --no-update` only sorted by name and not by name and
  version ([#5446](https://github.com/python-poetry/poetry/pull/5446))
- Fixed an issue where the Solver fails when a dependency has multiple constrained dependency definitions for the same
  package ([#5403](https://github.com/python-poetry/poetry/pull/5403))
- Fixed an issue where dependency resolution takes a while because Poetry checks all possible combinations
  even markers are mutually exclusive ([#4695](https://github.com/python-poetry/poetry/pull/4695))
- Fixed incorrect version selector constraint ([#5500](https://github.com/python-poetry/poetry/pull/5500))
- Fixed an issue where `poetry lock --no-update` dropped
  packages ([#5435](https://github.com/python-poetry/poetry/pull/5435))
- Fixed an issue where packages were incorrectly grouped when
  exporting ([#5156](https://github.com/python-poetry/poetry/pull/5156))
- Fixed an issue where lockfile always updates when using private
  sources ([#5362](https://github.com/python-poetry/poetry/pull/5362))
- Fixed an issue where the solver did not account for selected package features ([#5305](https://github.com/python-poetry/poetry/pull/5305))
- Fixed an issue with console script execution of editable dependencies on Windows ([#3339](https://github.com/python-poetry/poetry/pull/3339))
- Fixed an issue where editable builder did not write PEP-610 metadata ([#5703](https://github.com/python-poetry/poetry/pull/5703))
- Fixed an issue where Poetry 1.1 lock files were incorrectly identified as not fresh ([#5458](https://github.com/python-poetry/poetry/pull/5458))

### Docs

- Updated plugin management commands ([#5450](https://github.com/python-poetry/poetry/pull/5450))
- Added the `--readme` flag to documentation ([#5357](https://github.com/python-poetry/poetry/pull/5357))
- Added example for multiple maintainers ([#5661](https://github.com/python-poetry/poetry/pull/5661))
- Updated documentation for issues [#4800](https://github.com/python-poetry/poetry/issues/4800), [#3709](https://github.com/python-poetry/poetry/issues/3709), [#3573](https://github.com/python-poetry/poetry/issues/3573), [#2211](https://github.com/python-poetry/poetry/issues/2211) and [#2414](https://github.com/python-poetry/poetry/pull/2414) ([#5656](https://github.com/python-poetry/poetry/pull/5656))
- Added `poetry.toml` note in configuration ([#5492](https://github.com/python-poetry/poetry/pull/5492))
- Add documentation for `poetry about`, `poetry help`, `poetrylist`, and the `--full-path` and `--all` options
  documentation ([#5664](https://github.com/python-poetry/poetry/pull/5664))
- Added more clarification to the `--why` flag ([#5653](https://github.com/python-poetry/poetry/pull/5653))
- Updated documentation to refer to PowerShell for Windows, including
  instructions ([#3978](https://github.com/python-poetry/poetry/pull/3978), [#5618](https://github.com/python-poetry/poetry/pull/5618))
- Added PEP 508 name requirement ([#5642](https://github.com/python-poetry/poetry/pull/5642))
- Added example for each section of pyproject.toml ([#5585](https://github.com/python-poetry/poetry/pull/5642))
- Added documentation for `--local` to fix issue [#5623](https://github.com/python-poetry/poetry/issues/5623) ([#5629](https://github.com/python-poetry/poetry/pull/5629))
- Added troubleshooting documentation for using proper quotation with
  ZSH ([#4847](https://github.com/python-poetry/poetry/pull/4847))
- Added information on git and basic http auth ([#5578](https://github.com/python-poetry/poetry/pull/5578))
- Removed ambiguity about PEP 440 and semver ([#5576](https://github.com/python-poetry/poetry/pull/5576))
- Removed Pipenv comparison ([#5561](https://github.com/python-poetry/poetry/pull/5561))
- Improved dependency group related documentation ([#5338](https://github.com/python-poetry/poetry/pull/5338))
- Added documentation for default directories used by Poetry ([#5391](https://github.com/python-poetry/poetry/pull/5301))
- Added warning about credentials preserved in shell history ([#5726](https://github.com/python-poetry/poetry/pull/5726))
- Improved documentation of the `readme` option, including multiple files and additional formats ([#5158](https://github.com/python-poetry/poetry/pull/5158))
- Improved contributing documentation ([#5708](https://github.com/python-poetry/poetry/pull/5708))
- Remove all references to `--dev-only` option ([#5771](https://github.com/python-poetry/poetry/pull/5771))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+- Fixed more `truthy-bool` mypy warnings ([#5667](https://github.com/python-poetry/poetry/pull/5667))
+- Fixed #3628 and #4702 by handling invalid distributions gracefully ([#5645](https://github.com/python-poetry/poetry/pull/5645))
+
+### Added
+
+- Added support for multiple-constraint direct origin dependencies with the same version ([#5715](https://github.com/python-poetry/poetry/pull/5715))
+- Added support for `bool` values for TLS certificate verification of repositories ([#5719](https://github.com/python-poetry/poetry/pull/5719)
+- Added PEP-610 metadata for editable builds ([#5703](https://github.com/python-poetry/poetry/pull/5703))
+- Added `virtualenvs.prompt` option to config ([#5606](https://github.com/python-poetry/poetry/pull/5606))
+- Added @radoering to list of maintainers 🎉 ([#5632](https://github.com/python-poetry/poetry/pull/5632))
```
Comment (Secrus) [inline CHANGELOG.md:30]: That actually added everyone from the core team as the `maintainers` list wasn't there before.

Comment (kasteph) [reply CHANGELOG.md:30]: ah yea I noticed that later as I was going down the commits list: https://github.com/python-poetry/poetry/pull/5725/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR55

**Review by finswimmer — DISMISSED** [review]
Thanks for preparing the changelog so quickly  @kasteph :tada: :+1: :1st_place_medal: 

I don't think it is necessary to mention version bumps of dependencies, unless they introduce (important) changes to poetry.

To complete this release, the version in `pyproject.toml` must be bumped as well.

**Comment by abn** [comment]
Let's lock core here too. This is because we have a few changes in the next release that might not work with this pre-release.

**Comment by sdispater** [comment]
We should define a standard order of change log sections that we stick with.

From the start of Poetry I always tried to stick to:

1. Added
2. Changed
3. Fixed
4. Docs

The rationale behind that is that I want to notify about new features and changes first.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,118 @@
...
 
+## 1.2.0b2 - 2022-06-07
+
+### Added
+
+- Added support for multiple-constraint direct origin dependencies with the same
+  version ([#5715](https://github.com/python-poetry/poetry/pull/5715))
+- Added support disabling TLS verification for custom package sources via `poetry config certificates.<repository>.cert false` ([#5719](https://github.com/python-poetry/poetry/pull/5719)
+- Added `virtualenvs.prompt` option to config ([#5606](https://github.com/python-poetry/poetry/pull/5606))
+- Added progress to `download_file` if in Indicator context ([#5451](https://github.com/python-poetry/poetry/pull/5451))
```
Comment (neersighted) [inline CHANGELOG.md:11]:
```suggestion
- Added progress indicator to `download_file` (used when downloading dists) ([#5451](https://github.com/python-poetry/poetry/pull/5451))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,118 @@
...
+- Added `--dry-run` to the `version` command ([#5603](https://github.com/python-poetry/poetry/pull/5603))
```
Comment (neersighted) [inline CHANGELOG.md:12]:
```suggestion
- Added `--dry-run` to `poetry version` command ([#5603](https://github.com/python-poetry/poetry/pull/5603))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,118 @@
...
+- Added `--why` to `poetry show` ([#5444](https://github.com/python-poetry/poetry/pull/5444))
+- Added support for single page (html) repositories ([#5517](https://github.com/python-poetry/poetry/pull/5517))
+- Added support for PEP 508 strings when adding
+  dependencies via the `add` command ([#5554](https://github.com/python-poetry/poetry/pull/5554))
```
Comment (neersighted) [inline CHANGELOG.md:16]:
```suggestion
  dependencies via `poetry add` command ([#5554](https://github.com/python-poetry/poetry/pull/5554))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,118 @@
...
+- Added `--no-cache` as a global option ([#5519](https://github.com/python-poetry/poetry/pull/5519))
+- Added cert retrieval for HTTP requests made by Poetry ([#5320](https://github.com/python-poetry/poetry/pull/5320))
+- Added `--skip-existing` to `publish` command ([#2812](https://github.com/python-poetry/poetry/pull/2812))
```
Comment (neersighted) [inline CHANGELOG.md:19]:
```suggestion
- Added `--skip-existing` to `poetry publish` ([#2812](https://github.com/python-poetry/poetry/pull/2812))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,118 @@
...
+- Added `--all-extras` to `install` command ([#5452](https://github.com/python-poetry/poetry/pull/5452))
```
Comment (neersighted) [inline CHANGELOG.md:20]:
```suggestion
- Added `--all-extras` to `poetry install` ([#5452](https://github.com/python-poetry/poetry/pull/5452))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,118 @@
...
+- Added new `poetry self` sub-commands to manage plugins and/or system environment packages, eg: keyring backends ([#5450](https://github.com/python-poetry/poetry/pull/5450))
+- Added new configuration (`installer.no-binary`) to allow selection of non-binary distributions when installing a dependency ([#5609](https://github.com/python-poetry/poetry/pull/5609))
+
+### Changed
+
+- Replaced `poetry plugin` commands are now deprecated, use the more generic `poetry self`
```
Comment (neersighted) [inline CHANGELOG.md:26]:
```suggestion
- Replaced `poetry plugin` commands are now deprecated in favor of the more generic `poetry self`
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,118 @@
 # Change Log
 
+## 1.2.0b2 - 2022-06-07
+
+### Added
+
+- Added support for multiple-constraint direct origin dependencies with the same
+  version ([#5715](https://github.com/python-poetry/poetry/pull/5715))
+- Added support disabling TLS verification for custom package sources via `poetry config certificates.<repository>.cert false` ([#5719](https://github.com/python-poetry/poetry/pull/5719)
+- Added `virtualenvs.prompt` option to config ([#5606](https://github.com/python-poetry/poetry/pull/5606))
```
Comment (neersighted) [inline CHANGELOG.md:10]:
```suggestion
- Added new configuration (`virtualenvs.prompt`) to customize the prompt of the Poetry-managed virtual environment ([#5606](https://github.com/python-poetry/poetry/pull/5606))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,118 @@
...
+- Added `--skip-existing` to `publish` command ([#2812](https://github.com/python-poetry/poetry/pull/2812))
+- Added `--all-extras` to `install` command ([#5452](https://github.com/python-poetry/poetry/pull/5452))
+- Added new `poetry self` sub-commands to manage plugins and/or system environment packages, eg: keyring backends ([#5450](https://github.com/python-poetry/poetry/pull/5450))
+- Added new configuration (`installer.no-binary`) to allow selection of non-binary distributions when installing a dependency ([#5609](https://github.com/python-poetry/poetry/pull/5609))
+
+### Changed
+
+- Replaced `poetry plugin` commands are now deprecated, use the more generic `poetry self`
+  commands ([#5450](https://github.com/python-poetry/poetry/pull/5450))
+- When creating new projects, Poetry no longer restricts README formats to `md` and `rst` ([#5357](https://github.com/python-poetry/poetry/pull/5357))
```
Comment (neersighted) [inline CHANGELOG.md:28]:
```suggestion
- When creating new projects, Poetry no longer restricts README extensions to `md` and `rst` ([#5357](https://github.com/python-poetry/poetry/pull/5357))
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,118 @@
...
+- Changed the provider to allow fallback to installed packages ([#5704](https://github.com/python-poetry/poetry/pull/5704))
+- Solver now correctly handles and prefers direct reference constraints (vcs, file etc.) over public version identifiers ([#5654](https://github.com/python-poetry/poetry/pull/5654))
+- Changed the build script behavior to create an ephemeral build environment when a build script is
+  specified ([#5401](https://github.com/python-poetry/poetry/pull/5401))
+- Improved performance when determining PEP-517 metadata from sources ([#5601](https://github.com/python-poetry/poetry/pull/5601))
+- Project package sources no longer require to be redefined as global repositories when configuring credentials ([#5563](https://github.com/python-poetry/poetry/pull/5563))
```
Comment (neersighted) [inline CHANGELOG.md:34]:
```suggestion
- Project package sources no longer need to be redefined as global repositories when configuring credentials ([#5563](https://github.com/python-poetry/poetry/pull/5563))
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,118 @@
...
+- Replaced system git command use with dulwich, in order to force the legacy behaviour set `experimental.system-git-client` configuration to `true` ([#5428](https://github.com/python-poetry/poetry/pull/5428))
```
Comment (neersighted) [inline CHANGELOG.md:35]:
```suggestion
- Replaced external git command use with dulwich, in order to force the legacy behaviour set `experimental.system-git-client` configuration to `true` ([#5428](https://github.com/python-poetry/poetry/pull/5428))
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,118 @@
...
+- Improved http request handling for sources and multiple paths on same netloc ([#5518](https://github.com/python-poetry/poetry/pull/5518))
+- Made `no-pip` and `no-setuptools` configuration explicit ([#5455](https://github.com/python-poetry/poetry/pull/5455))
+- Improved application logging, use of `-vv` now provides more debug information ([#5503](https://github.com/python-poetry/poetry/pull/5503))
+- Renamed implicit group `default` to `main` ([#5465](https://github.com/python-poetry/poetry/pull/5465))
+- Replaced in-tree implementation
```
Comment (neersighted) [inline CHANGELOG.md:40]:
```suggestion
- Replaced in-tree implementation of `poetry export`
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,5 +1,118 @@
...
+  with `poetry-plugin-export` ([#5413](https://github.com/python-poetry/poetry/pull/5413))
+- Changed the password manager behavior to use a `"null"` keyring when
+  disabled ([#5251](https://github.com/python-poetry/poetry/pull/5251))
+- Incremental improvement for Solver performance ([#5335](https://github.com/python-poetry/poetry/pull/5335))
```
Comment (neersighted) [inline CHANGELOG.md:44]:
```suggestion
- Incremental improvement of Solver performance ([#5335](https://github.com/python-poetry/poetry/pull/5335))
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,5 +1,118 @@
...
+- Virtual environments created on macOS now are excluded from Time Machine backups ([#4599](https://github.com/python-poetry/poetry/pull/4599))
```
Comment (neersighted) [inline CHANGELOG.md:45]:
```suggestion
- Newly created virtual environments on macOS now are excluded from Time Machine backups ([#4599](https://github.com/python-poetry/poetry/pull/4599))
```

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,5 +1,118 @@
...
+- Update `packaging` dependency to use major version 21, this change forces Poetry to drop support for managing Python 2.7 environments ([#4749](https://github.com/python-poetry/poetry/pull/4749))
+
+### Fixed
+
+- Fixed `poetry update --dry-run` to not modify `poetry.lock` ([#5718](https://github.com/python-poetry/poetry/pull/5718)
+  , [#3666](https://github.com/python-poetry/poetry/issues/3666)
+  , [#3766](https://github.com/python-poetry/poetry/issues/3766))
+- Fixed [#5537](https://github.com/python-poetry/poetry/issues/5537) where export fails to resolve dependencies with more than one
+  path ([#5688](https://github.com/python-poetry/poetry/pull/5688))
+- Fixed an issue where environment variables, `POETRY_CONFIG_DIR` and `POETRY_CACHE_DIR`, were not being respected (
```
Comment (neersighted) [inline CHANGELOG.md:56]:
```suggestion
- Fixed an issue where the environment variables `POETRY_CONFIG_DIR` and `POETRY_CACHE_DIR` were not being respected (
```

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,5 +1,118 @@
...
+  sources ([#5362](https://github.com/python-poetry/poetry/pull/5362))
+- Fixed an issue where the solver did not account for selected package features ([#5305](https://github.com/python-poetry/poetry/pull/5305))
+- Fixed an issue with console script execution of editable dependencies on Windows ([#3339](https://github.com/python-poetry/poetry/pull/3339))
+- Fixed an issue where editable builder did not write PEP-610 metadata ([#5703](https://github.com/python-poetry/poetry/pull/5703))
+- Fixed an issue where Poetry 1.1 lock files were incorrectly identified as not fresh ([#5458](https://github.com/python-poetry/poetry/pull/5458))
+
+### Docs
+
+- Updated plugin management commands ([#5450](https://github.com/python-poetry/poetry/pull/5450))
+- Added `--readme` flag to documentation ([#5357](https://github.com/python-poetry/poetry/pull/5357))
```
Comment (neersighted) [inline CHANGELOG.md:90]:
```suggestion
- Added the `--readme` flag to documentation ([#5357](https://github.com/python-poetry/poetry/pull/5357))
```

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,5 +1,118 @@
...
+- Added example for multiple maintainers ([#5661](https://github.com/python-poetry/poetry/pull/5661))
+- Updated documentation for issues #4800, #3709, #3573, #2211 and PR
+  #2414 ([#5656](https://github.com/python-poetry/poetry/pull/5656))
+- Added `poetry.toml` note in configuration ([#5492](https://github.com/python-poetry/poetry/pull/5492))
+- CLI: Added `about`, `help`, `list`, `--full-path`, `--all`
```
Comment (neersighted) [inline CHANGELOG.md:95]:
```suggestion
- Add documentation for `poetry about`, `poetry help`, `poetrylist`, and the `--full-path` and `--all` options
```

File: CHANGELOG.md [hunk 18]
```diff
@@ -1,5 +1,118 @@
...
+  documentation ([#5664](https://github.com/python-poetry/poetry/pull/5664))
+- Added more clarification to the `--why` flag ([#5653](https://github.com/python-poetry/poetry/pull/5653))
+- Updated documentation to refer to PowerShell for Windows, incl.
```
Comment (neersighted) [inline CHANGELOG.md:98]:
```suggestion
- Updated documentation to refer to PowerShell for Windows, including
```

File: CHANGELOG.md [hunk 19]
```diff
@@ -1,5 +1,118 @@
...
+  instructions ([#3978](https://github.com/python-poetry/poetry/pull/3978)
+  , [#5618](https://github.com/python-poetry/poetry/pull/5618))
+- Added PEP-508 name requirement ([#5642](https://github.com/python-poetry/poetry/pull/5642))
```
Comment (neersighted) [inline CHANGELOG.md:101]:
```suggestion
- Added PEP 508 name requirement ([#5642](https://github.com/python-poetry/poetry/pull/5642))
```

File: CHANGELOG.md [hunk 20]
```diff
@@ -1,5 +1,118 @@
...
+- Added example for each section of pyproject.toml ([#5585](https://github.com/python-poetry/poetry/pull/5642))
+- Added documentation for `--local` to fix issue #5623 ([#5629](https://github.com/python-poetry/poetry/pull/5629))
+- Added troubleshooting documentation for using proper quotation with
+  ZSH ([#4847](https://github.com/python-poetry/poetry/pull/4847))
+- Added information on git and basic http auth ([#5578](https://github.com/python-poetry/poetry/pull/5578))
+- Removed ambiguity about pep440 and semver ([#5576](https://github.com/python-poetry/poetry/pull/5576))
```
Comment (neersighted) [inline CHANGELOG.md:107]:
```suggestion
- Removed ambiguity about PEP 440 and semver ([#5576](https://github.com/python-poetry/poetry/pull/5576))
```

File: CHANGELOG.md [hunk 21]
```diff
@@ -1,5 +1,118 @@
...
+- Removed Pipenv comparison ([#5561](https://github.com/python-poetry/poetry/pull/5561))
+- Improved dependency group related documentation ([#5338](https://github.com/python-poetry/poetry/pull/5338))
+- Added documentation for default directories used by Poetry ([#5391](https://github.com/python-poetry/poetry/pull/5301))
+- Added warning about credentials preserved in shell history ([#5726](https://github.com/python-poetry/poetry/pull/5726))
+- Improved documentation for project readme file configuration for multiple files and additional formats ([#5158](https://github.com/python-poetry/poetry/pull/5158))
```
Comment (neersighted) [inline CHANGELOG.md:112]:
```suggestion
- Improved documentation of the `readme` option, including multiple files and additional formats ([#5158](https://github.com/python-poetry/poetry/pull/5158))
```

**Review by neersighted — CHANGES_REQUESTED** [review]

**Review by neersighted — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 22]
```diff
@@ -1,5 +1,118 @@
...
+- Added `--no-cache` as a global option ([#5519](https://github.com/python-poetry/poetry/pull/5519))
+- Added cert retrieval for HTTP requests made by Poetry ([#5320](https://github.com/python-poetry/poetry/pull/5320))
+- Added `--skip-existing` to `poetry publish` ([#2812](https://github.com/python-poetry/poetry/pull/2812))
+- Added `--all-extras` to `poetry install` ([#5452](https://github.com/python-poetry/poetry/pull/5452))
+- Added new `poetry self` sub-commands to manage plugins and/or system environment packages, eg: keyring backends ([#5450](https://github.com/python-poetry/poetry/pull/5450))
+- Added new configuration (`installer.no-binary`) to allow selection of non-binary distributions when installing a dependency ([#5609](https://github.com/python-poetry/poetry/pull/5609))
+
+### Changed
+
+- Replaced `poetry plugin` commands are now deprecated in favor of the more generic `poetry self`
```
Comment (neersighted) [inline CHANGELOG.md:26]:
```suggestion
- `poetry plugin` commands are now deprecated in favor of the more generic `poetry self`
```

File: CHANGELOG.md [hunk 23]
```diff
@@ -1,5 +1,118 @@
...
+### Changed
+
+- `poetry plugin` commands are now deprecated in favor of the more generic `poetry self`
+  commands ([#5450](https://github.com/python-poetry/poetry/pull/5450))
+- When creating new projects, Poetry no longer restricts README extensions to `md` and `rst` ([#5357](https://github.com/python-poetry/poetry/pull/5357))
+- Changed the provider to allow fallback to installed packages ([#5704](https://github.com/python-poetry/poetry/pull/5704))
+- Solver now correctly handles and prefers direct reference constraints (vcs, file etc.) over public version identifiers ([#5654](https://github.com/python-poetry/poetry/pull/5654))
+- Changed the build script behavior to create an ephemeral build environment when a build script is
+  specified ([#5401](https://github.com/python-poetry/poetry/pull/5401))
+- Improved performance when determining PEP-517 metadata from sources ([#5601](https://github.com/python-poetry/poetry/pull/5601))
```
Comment (neersighted) [inline CHANGELOG.md:33]:
```suggestion
- Improved performance when determining PEP 517 metadata from sources ([#5601](https://github.com/python-poetry/poetry/pull/5601))
```

**Review by abn — APPROVED** [review]

**Pull request merged**